templates/sssd-session-recording.conf

```diff
@@ -1,4 +1,4 @@
 [session_recording]
 scope={{ tlog_scope_sssd }}
-users={{ tlog_users_sssd|join(', ') }}
-groups={{ tlog_groups_sssd|join(', ') }}
+users={{ tlog_users_sssd }}
```

Comment:
I don't think this is correct - I think the current code is correct

Reply:
agreed but, could the code be more flexible to avoid this issue?

something like this maybe?  (I was hoping for a better (more ansible) way but, this is what I came up with on short notice.  there may be other flaws there)


`{% if tlog_users_sssd is string %}
users={{ tlog_users_sssd }}
{% else %}
users={{ tlog_users_sssd|join(', ') }}
{% endif %}
{% if tlog_groups_sssd is string %}
groups={{ tlog_groups_sssd }}
{% else %}
groups={{ tlog_groups_sssd|join(', ') }}
{% endif %}
`

Reply:
that works - for a one liner you could use
```
users={{ tlog_users_sssd if tlog_users_sssd is string else tlog_users_sssd|join(', ') }}
```
and similarly for groups

To make this change, you would also need a corresponding change to the documentation, and a new test to test users (and groups) that can have a string or a list value